es6-shim.js

```diff
@@ -205,7 +205,15 @@
     return result;
   };
 
+  var bindContext = function (func, context) {
```

Comment:
`var bindContext = Function.call.bind(Function.apply)`? I'm not even sure we need a shortcut variable defined here.